index.js

```diff
@@ -13,5 +13,20 @@ module.exports = (robot, _, Settings = require('./lib/settings')) => {
     if (defaultBranch && settingsModified) {
       return Settings.sync(context.github, context.repo())
     }
+    else if (!defaultBranch && settingsModified) {
+      let settingChangeCommits = payload.commits.filter(commit => {  //get all commits that change settings.yml
+        return commit.added.includes(Settings.FILE_NAME) || 
+        commit.modified.includes(Settings.FILE_NAME)
+      })
+      
+      settingChangeCommits.forEach(function(commit, index, array) { //create status for each commit that changed settings.yml
```

Comment:
use ```async function (commit, index, array){```

Reply:
doing this helps the test to pass but it still says that `createStatus` is undefined. I don't know why. I've used the same thing in my other app and that works fine.